### DISCUSSION TITLE
WSL installation branch for Windows

### DISCUSSION THREAD

**Discussion by Jupiteur** [pystardust/ani-cli#267] [Ideas] [answered]
The WSL (Windows Subsystem for Linux) installs a ligthweigth linux OS
It can execute Windows apps from it with their name (vlc.exe) if they are in the Windows path or with the absolute path (/mnt/c/Program\ Files/VideoLAN/VLC/vlc.exe) if not

**Comment by port19x** [comment, marked as answer]
Shouldn't be necessary anymore
